perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -124,7 +133,7 @@ def _Delete(self):
     deleted.
     """
     cmd = util.GcloudCommand(self, 'sql', 'instances', 'delete',
-                             self.instance_id)
+                             self.instance_id, '--quiet')
```

Comment:
I don't think so. 
From gcloud docs: 
```
--quiet, -q
Disable all interactive prompts when running gcloud commands.
If input is required, defaults will be used, or an error will be raised.
Overrides the default core/disable_prompts property value for this command invocation.
```
Further stdout is directed in the[ util command](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/vm_util.py#L293). 

And I noted that we did the same for [mysql_service_benchmark](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py#L841).